logging/resource-logs-viewing.adoc

```diff
@@ -0,0 +1,23 @@
+:context: resource-logs-viewing
+[id="resource-logs-viewing"]
+= Viewing logs for a resource
+include::modules/common-attributes.adoc[]
```

Comment:
Looks like all of the correct metadata is present here, but needs to be reordered.

[Assembly file metadata](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata)